MiniApp/Classes/MiniAppDownloader.swift

```diff
@@ -27,7 +27,7 @@ class MiniAppDownloader {
             switch result {
             case .success(let responseData):
                 self.completionHandler = completionHandler
-                self.downloadManifestFiles(with: appId, versionId: versionId, files: responseData.manifest)
+                self.downloadManifestFiles(with: appId, versionId: versionId, files: responseData.files)
```

Comment:
As per the latest API specs, it will be `manifest`

Reply:
Can you use codingKeys so that we can call it what we want internally and if the backend keeps changing the naming manifest/files/whatever it won't affect the SDK?